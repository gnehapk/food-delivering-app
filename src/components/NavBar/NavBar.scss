.nav-bar {
  display: flex;
  justify-content: space-between;
  height: 60px;
  padding: 0 4rem;

  .user-section {
    padding-top: 1.2rem;
    .username {
      display: none;

      @media only screen and (min-width: 768px) {
        display: inline;
        vertical-align: middle;
        padding-left: 0.4rem;
      }
    }
  }
  
}
