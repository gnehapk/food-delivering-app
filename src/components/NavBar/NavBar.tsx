import React from 'react';
import logo from '../../assets/logo-horizontal.svg';
import username from  '../../assets/user-icon.svg';

import './NavBar.scss';

const NavBar: React.FC = () => (
    <div className="nav-bar">
        <img alt="logo" src={logo} />
        <div className="user-section">
          <img alt="username" src={username} />
          <span className="username">Jane Smith</span>
        </div>
    </div>
);

export default NavBar;