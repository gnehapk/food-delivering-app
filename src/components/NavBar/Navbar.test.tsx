import React from 'react';
import TestRenderer from 'react-test-renderer';
import NavBar from './NavBar';

describe('NavBar', () => {
  it('should render correctly', () => {
    const tree = TestRenderer.create(<NavBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

