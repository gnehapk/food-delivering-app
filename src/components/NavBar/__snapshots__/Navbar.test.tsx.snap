// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should render correctly 1`] = `
<div
  className="nav-bar"
>
  <img
    alt="logo"
    src="logo-horizontal.svg"
  />
  <div
    className="user-section"
  >
    <img
      alt="username"
      src="user-icon.svg"
    />
    <span
      className="username"
    >
      Jane Smith
    </span>
  </div>
</div>
`;
